content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -0,0 +1,101 @@
+---
+reviewers:
+title: RBAC Good Practices
+description: >
+  Principles and practices for good RBAC design for cluster operators.
+content_type: concept
+weight: 10
+---
+
+# Kubernetes RBAC Good Practices
```

Comment:
Please omit this; the Hugo theme that we use renders a first-level heading based on the title you set in the front matter.

Reply:
removed